### PR TITLE
vpat 26: quickformat more detailed instructions

### PR DESCRIPTION
**Pull request by abaevbog**
I think this makes it a bit clearer as to what exactly is going on in the QF and what options are available.

Fixes: #4002

### CODE REVIEW

File: chrome/locale/en-US/zotero/zotero.ftl [hunk 1]
```diff
@@ -468,9 +468,15 @@ toggle-preview =
...
-quickformat-aria-input = Type your search term. Use Down Arrow and Up Arrow to navigate the search results.
-quickformat-aria-item = Press { return-or-enter } to select this item. Press Tab to return to the search field.
+quickformat-general-instructions = Use Left/Right arrows to navigate the entries of this citation. {
+    $dialogMenu ->
+        [active] Press Shift-Tab to focus the dialog's menu.
+        *[other] { "" }
+} Press { return-or-enter } to save edits to this citation. Press Escape to discard the changes and close the dialog.
+
+quickformat-aria-bubble = Item included in this citation. Press Down Arrow to open citation properties. { quickformat-general-instructions }
+quickformat-aria-input = Citation item search input. Type to search for an item to include in this citation. Then Press Tab to focus and navigate the list of matching items to add to this citation. { quickformat-general-instructions }
```
Comment (adomasven) [inline chrome/locale/en-US/zotero/zotero.ftl:478]: Maybe "Then Press Tab to focus and navigate the list of search results to add to this citation."?

**Comment by adomasven** [comment]
> The citation selection process should provide clear, step-by-step instructions guiding users on how to search for, select, modify, and confirm citations, as well as how to add multiple citations if desired.

I didn't test this with a screen reader, but is "open citation properties" clear enough regarding "instructions on how to [...] modify [...] citations"? I guess that popover just has fields that are pretty self-explanatory?

File: chrome/locale/en-US/zotero/zotero.ftl (hunk 1 above)
Comment (abaevbog) [reply chrome/locale/en-US/zotero/zotero.ftl]: Sure, that sounds nice

**Comment by abaevbog** [comment]
> I didn't test this with a screen reader, but is "open citation properties" clear enough regarding "instructions on how to [...] modify [...] citations"? I guess that popover just has fields that are pretty self-explanatory?

I changed it for "Item included in this citation. Press Down Arrow to open a dialog with citation properties to add details. " - that should indicate as to "why" you would want to open the popup. Also, I added the aria properties to make it clear that the properties popover is a dialog.

And then, yeah, as soon as it is opened, the "Page" (or whatever locator is selected) input is focused, and it gets announced as such, so it should be fairly clear (or at least clearer) what can be done.

**Pull request merged**